README.md

```diff
@@ -55,55 +68,54 @@ The following steps are required to deploy the infrastructure from the command l
 
 1. Create a resource group and deploy the infrastructure.
 
-   Make sure:
-
-   - The BASE_NAME contains only lowercase letters and is between 6 and 8 characters. Most resource names will include this text.
-   - You choose a valid resource group name.
-
    ```bash
    LOCATION=eastus
-   BASE_NAME=<base-resource-name (between 6 and 8 lowercase characters)>
+   BASE_NAME=<base resource name, between 6 and 8 lowercase characters, most resource names will include this text>
 
-   RESOURCE_GROUP=<resource-group-name>
+   RESOURCE_GROUP=rg-chat-basic-${LOCATION}
```

Comment:
Changing what?  The user provided their own value before. This just gives them a template to work with.

Reply:
I'm still not following. What does resource group name have anything to do with multiple customers?

Reply:
In the old solution 2 customers would likely choose unique names for their respective RGs. Given this update, customer one deploying in eastus would deploy a RG called rg-chat-basic-eastus and it would work. Customer two would try to deploy a RG in eastus named again rg-chat-basic-eastus and it would fail.
Hope that helps and this is not a big deal - just a thought. Feel free to disregard and self resolve the conversation.